std/assembly/index.d.ts

```diff
@@ -1052,13 +1052,13 @@ declare type returnof<T extends (...args: any) => any> = ReturnType<T>;
 /** Pseudo-class representing the backing class of integer types. */
 declare class _Integer {
   /** Smallest representable value. */
-  static readonly MIN_VALUE: number;
+  static readonly MIN_VALUE: i8 | u8 | i16 | u16 | i32 | u32 | i64 | u64;
   /** Largest representable value. */
-  static readonly MAX_VALUE: number;
+  static readonly MAX_VALUE: i8 | u8 | i16 | u16 | i32 | u32 | i64 | u64;
   /** Converts a string to an integer of this type. */
-  static parseInt(value: string, radix?: number): number;
+  static parseInt(value: string, radix?: number): i8 | u8 | i16 | u16 | i32 | u32 | i64 | u64;
```

Comment:
These look like we could either just leave them `number`, or copy them to each respective class with the proper type for clarity?

Reply:
How about create:
```ts
type anyint = i8 | u8 | i16 | u16 | i32 | u32 | i64 | u64;
type anyreal = f32 | f64;
```
And use this compound types?
?

Reply:
However it seems TS fold all this to just `number` during suggestions. Ok I'll revert this changes